tests/ssl/containers/array/test.swift

```diff
@@ -117,4 +117,163 @@ func test_add2() {
   sink(sunk : arr2[0]); //!testing!sink
 }
 
+// ------- Removing -------
+
+func test_removeAt() {
+  let src = source();
+  var arr = ["a", src, "b", "c"]; //!testing!source
+  let removed = arr.remove(at: 2);
+  sink(sunk : removed); //!testing!sink
+}
+
+func test_removeFirst() {
+  let src = source();
+  var arr = ["a", src, "b", "c"]; //!testing!source
+  let removed = arr.removeFirst();
+  sink(sunk : removed); //!testing!sink
+}
+
+func test_removeLast() {
+  let src = source();
+  var arr = ["a", src, "b", "c"]; //!testing!source
+  let removed = arr.removeLast();
+  sink(sunk : removed); //!testing!sink
+}
+
+func test_removeAll(){
+  let src = source();
+  var arr = ["a", src, "b", "c"]; //!testing!source
+  arr.removeAll();
+  sink(sunk : arr[0]); //!testing!sink!fp
```

Comment:
Add `SWAN-27` to the comment so that we know what bug relates to this false positive.